cristin-person/src/test/java/no/unit/nva/cristin/person/institution/fetch/FetchPersonInstitutionInfoHandlerTest.java

```diff
@@ -0,0 +1,180 @@
+import static no.unit.nva.cristin.person.institution.fetch.FetchPersonInstitutionInfoHandler.PERSON_ID;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static nva.commons.apigateway.MediaTypes.APPLICATION_PROBLEM_JSON;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.net.HttpHeaders;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.util.Map;
+import no.unit.nva.cristin.person.model.cristin.CristinPersonInstitutionInfo;
+import no.unit.nva.cristin.person.model.nva.PersonInstitutionInfo;
+import no.unit.nva.cristin.testing.HttpResponseFaker;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.GatewayResponse;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.paths.UriWrapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FetchPersonInstitutionInfoHandlerTest {
+
+    private static final String EMPTY_JSON = "{}";
+    private static final String VALID_PERSON_ID = "12345";
+    private static final String VALID_UNIT_ID = "185.90.0.0";
+    private static final String VALID_INSTITUTION_ID = "185";
+    private static final String EXPECTED_CRISTIN_URI =
+        "https://api.cristin-test.uio.no/v2/persons/12345/institutions/185";
+    private static final String INVALID_PATH_PARAM = "abcdef";
+    private static final URI EXPECTED_ID_URI = getExpectedId();
+
+    private final HttpClient clientMock = mock(HttpClient.class);
+    private final Environment environment = new Environment();
+    private FetchPersonInstitutionInfoClient apiClient;
+    private Context context;
+    private ByteArrayOutputStream output;
+    private FetchPersonInstitutionInfoHandler handler;
+
+    @BeforeEach
+    void setUp() throws IOException, InterruptedException {
+        when(clientMock.<String>send(any(), any())).thenReturn(new HttpResponseFaker(EMPTY_JSON, 200));
+        apiClient = new FetchPersonInstitutionInfoClient(clientMock);
+        context = mock(Context.class);
+        output = new ByteArrayOutputStream();
+        handler = new FetchPersonInstitutionInfoHandler(apiClient, environment);
+    }
+
+    @Test
+    void shouldReturnCristinResponseMappedToNvaFormatWhenCallingEndpointWithValidParams()
```

Comment:
I think the description of this test is very generic. if you are testing the absence of query parameters then test should be called that. If you are testing the response then define what is the valid query.